create.sql

```diff
@@ -409,7 +423,7 @@ CREATE TABLE gem_user (
     first character varying(20) NOT NULL,
     last character varying(20) NOT NULL,
     md5 character(32) NOT NULL,
-    email character varying(40) NOT NULL,
+    email character varying(40),
```

Comment:
I'm not sure how or why the email constraint disappeared.  I assume it must have to do with the two of us editing the database concurrently and doing `pg_dump`.  If so, it will continue to be an issue I guess.